Documentation/user-guides/getting-started.md

```diff
@@ -0,0 +1,148 @@
+# Prometheus Operator
+
+The mission of the Prometheus Operator is to make running Prometheus on top of Kubernetes as easy as possible, while preserving configurability as well as making the configuration Kubernetes native.
+
+To follow this getting started you will need a Kubernetes cluster you have access to. Let's give the Prometheus Operator a spin:
+
+[embedmd]:# (../../deployment.yaml)
+```yaml
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: prometheus-operator
+  labels:
+    operator: prometheus
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        operator: prometheus
+    spec:
+      containers:
+       - name: prometheus-operator
+         image: quay.io/coreos/prometheus-operator:v0.6.0
+         resources:
+           requests:
+             cpu: 100m
+             memory: 50Mi
+           limits:
+             cpu: 200m
+             memory: 100Mi
+```
+
+The Prometheus Operator introduces third party resources in Kubernetes to declare the desired state of a Prometheus and Alertmanager cluster as well as the Prometheus configuration. The resources it introduces are:
+
+* [`Prometheus`](../prometheus.md)
+* [`Alertmanager`](../alertmanager.md)
+* [`ServiceMonitor`](../service-monitor.md)
+
+> Important for this guide are the `Prometheus` and `ServiceMonitor` resources. Have a look at the [alerting guide](alerting.md) for more information about the `Alertmanager` resource.
+
+The Prometheus resource includes fields such as the desired version of Prometheus to run, the number of replicas, as well as a number of parameters to configure Prometheus itself. The connection of the Prometheus resource to the `ServiceMonitor` is established through the `serviceMonitorSelector`, which selects which `ServiceMonitor`s are to be used to generate the configuration file for Prometheus.
+
+We will walk through an example application that could look like this:
```

Comment:
Replace occurrences of "could look like this" with a brief description of what it actually is or what we are doing.

Here maybe: "First, we deploy three instances of a simple example application that listenes and exposes metrics on port 8080"